test/client/DummyServer.ts

```diff
@@ -108,105 +112,105 @@ export class DummyServer extends Server {
 
         const listKnownPackagesResponse = new ListKnownPackagesResponse();
 
-        this.addService(ActiveContractsServiceService, {
+        this.addService((activeContractsService as any)['com.daml.ledger.api.v1.ActiveContractsService'], {
```

Comment:
This is the most annoying part on the server side. You no longer get the export of the service directly but only the untyped map. The code here matches the example in https://github.com/agreatfool/grpc_tools_node_protoc_ts/blob/af23508a32b19e6a65f41a01cd1096bb4810a783/examples/src/grpcjs/server.ts

Reply:
Note that the whole server side is only relevant for tests.